windows/driver/src/faultinj/ex.c

```diff
@@ -1,65 +0,0 @@
-///
```

Comment:
Can this commit just be merged with https://github.com/S2E/guest-tools/pull/17/commits/95b3ddb9b8a2f606c50785a326f10a12fc8077f4 ?

I don't see the point of introducing this ex.c file and then the next commit that deals with faultinj replaces it with ex.cpp